setup.py

```diff
@@ -225,6 +226,40 @@ def run(self):
             sdist.run(self)
 
 
+def get_build_overrides():
+    """
+    Custom build commands to add `--std=c99` to compilation
+    """
+    from numpy.distutils.command.build_clib import build_clib
+    from numpy.distutils.command.build_ext import build_ext
+
+    def _is_using_gcc(obj):
+        is_gcc = False
+        if obj.compiler.compiler_type == 'unix':
+            cc = sysconfig.get_config_var("CC")
```

Comment:
Shouldn't we be looking at `obj.compiler` to learn this, rather than global state that may not be equivalent?

Reply:
Distutils itself does it like this. But I'm not sure if there is some reason why.

Reply:
once burned twice shy: I prefer to mimic distutils as far as possible